python_modules/dagit/dagit_tests/graphql/test_misc.py

```diff
@@ -747,6 +760,119 @@ def has_event_of_type(logs, message_type):
 }
 '''
 
+START_PIPELINE_EXECUTION_SNAPSHOT_QUERY = '''
```

Comment:
consider moving these in to their own test

Reply:
not following. do you mean test file?